profiles/serializer/normalizer.py

```diff
@@ -17,34 +20,40 @@ def normalize_snippet(value: Any) -> Any:
 @normalize.register(Profile)
 def normalize_profile(profile: Profile) -> dict:
     data = normalize_profile_snippet(profile)
-    data['emailAddresses'] = [normalize(email) for email in profile.get_email_addresses()]
-    data['affiliations'] = [normalize(aff) for aff in profile.get_affiliations()]
+    data['emailAddresses'] = [normalize(email) for email in profile.get_email_addresses(True)]
+    data['affiliations'] = [normalize(aff) for aff in profile.get_affiliations(True, True)]
 
     return data
 
 
 @normalize.register(Affiliation)
 def normalize_affiliation(affiliation: Affiliation) -> dict:
     return {
-        "name": [affiliation.department, affiliation.organisation],
-        "address": {
-            "formatted": [
-                affiliation.address.city,
-                affiliation.address.region,
-                affiliation.address.country.name,
-            ],
-            "components": {
-                "locality": [affiliation.address.city],
-                "area": [affiliation.address.region],
-                "country": affiliation.address.country.name
+        'access': ACCESS_RESTRICTED if affiliation.restricted else ACCESS_PUBLIC,
```

Comment:
being this *only* the schema change (and having no authentication yet), shouldn't we include *only* the public data, and separately added authentication to expose `restricted` values?
The authentication works with a combination of magic https://github.com/elifesciences/builder-private/blob/master/pillar/api-gateway.sls#L160 and more magic https://github.com/elifesciences/lax-formula/blob/master/salt/lax/config/etc-nginx-sitesavailable-lax.conf#L18-L25 so will need a review and genericization

Reply:
We'll do a follow-up to expose `restricted` values as nothing will use them (yet).